packages/lit-protocol-cipher/src/lit-protocol-cipher-provider.ts

```diff
@@ -50,17 +43,7 @@ export default class LitProvider implements CipherProviderTypes.ICipherProvider
   /**
    * @property {LitNodeClient|LitNodeClientNodeJs|null} client - The Lit Protocol client instance.
    */
-  private client: LitNodeClient | LitNodeClientNodeJs | null = null;
-
-  /**
-   * @property {any} storageProvider - The storage provider for the Node.js Lit client.
-   */
-  private nodeJsStorageProvider: StorageProvider | undefined;
-
-  /**
-   * @property {boolean} debug - A boolean indicating if debug mode is enabled.
-   */
-  private debug: boolean;
+  private litClient: LitNodeClient | LitNodeClientNodeJs | null = null;
```

Comment:
Since the `litClient` is passed in to the constructor, I don't think it needs to be `nullable`.
```suggestion
  private litClient: LitNodeClient | LitNodeClientNodeJs;
```